### DISCUSSION TITLE
Where is the code that adjusts the registry? (Windows 10)

### DISCUSSION THREAD

**Discussion by samuelwhiskeyjohnson** [opensteno/plover#1675] [Technical Discussion]
![Plover Registery](https://github.com/openstenoproject/plover/assets/71688370/c4401bdb-6e8f-4f9d-a8c1-f33c4d469c55)
Plover changes value of LowLevelHooksTimeout. I wonder where the code that does that? I can't find it in plover/registry.py

**Comment by user202729** [comment]
It's in `plover/oslayer/windows/keyboardcontrol.py` using `winreg` Python library

By the way, Plover architecture is (partially) documented in https://plover.readthedocs.io/en/latest/api/registry.html . In particular plover.registry module is to maintain the **plugin registry** of Plover, and has nothing to do with Windows registry.
